src/api/comments.ts

```diff
@@ -0,0 +1,31 @@
+import { BASE_URL, request } from './api';
+import { Comment } from '../types/Comment';
+
+export const getPostComments = (postId: number): Promise<Comment[]> => {
+  return request(`${BASE_URL}/comments`)
+    .then(comments => comments.filter((comment: Comment) => comment.postId === postId));
```

Comment:
use this ability of API to load needed comments 
<img width="600" alt="image" src="https://user-images.githubusercontent.com/64437769/158171860-d2954135-ab51-4fcc-8c8f-0ab80ad57e05.png">